.github/files/build-all-projects.sh

```diff
@@ -178,6 +178,32 @@ for SLUG in "${SLUGS[@]}"; do
 	# Copy SECURITY.md
 	cp "$BASE/SECURITY.md" "$BUILD_DIR/SECURITY.md"
 
+	if [[ "$SLUG" == "plugins/jetpack" ]]; then
```

Comment:
We'd also need this for Backup for sure and possibly the other plugins—even VaultPress. Once this works for Jetpack, though, we can add the others later.